index.js

```diff
@@ -236,6 +241,43 @@ var slackWebhooks = function() {
 
 	}
 
+	/**
+	 * Recursively searches an error for the stack trace.
+	 *
+	 * @function _findStackTrace
+	 *
+	 * @since 0.1.1
+	 * @access private
+	 * @memberOf console-to-slack
+	 * @param {object}   error  The error object.
+	 * @returns {string} stack  The stack trace from the error.
+	 */
+	function _findStackTrace(error) {
```

Comment:
I think it's fine to leave it as findStrackTrace until we have a use case for more abstraction -- and because it's a private method, it should only be called internally so there's no issue with changing APIs and legacy support.